ui/src/app/shared/services/applications-service.ts

```diff
@@ -355,9 +355,7 @@ export class ApplicationsService {
         const search = new URLSearchParams();
         search.set('container', containerName);
         search.set('namespace', namespace);
-        if (follow) {
-            search.set('follow', follow.toString());
-        }
+        search.set('follow', follow.toString());
```

Comment:
why this change is required? actually with this i think you will not be able to disable follow mode, no ?

Reply:
After a recent [grpc upgrade](https://github.com/argoproj/argo-cd/pull/8929) some previously-optional fields became required by the API. I haven't dug into precisely why they became required.

I don't think this change will make it impossible to disable follow mode. It just ensures that whatever `follow` value is set (true or false), it's encoded in the query string. I'll test locally to be sure.

Reply:
@pasha-codefresh I've confirmed that follow/non-follow still works. I've also pushed a couple more `group || ''` changes. I discovered.